.travis.yml

```diff
@@ -34,7 +34,7 @@ install:
   - conda info -a
   - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION numpy pytest cython nose boto3 PyYAML Click pytest numba
   - source activate test-environment
-  - pip install glob2
+  - pip install glob2 pylint tornado
```

Comment:
It was not installed before but we seemed to use it in cloudpickle.py[1] if it was available. The problem is `lint` doesn't like it when we import packages in `_rebuild_tornado_coroutine`. 


[1] https://github.com/pywren/pywren/blob/52b0e524ae3226da530f4722bf9f5a6abbde7250/pywren/serialize/cloudpickle/cloudpickle.py#L688